File: C#/BluffinMuffin.Protocol/Game/PlayerSitOutResponse.cs
namespace BluffinMuffin.Protocol.Game
{
    /// <summary>
    /// The response from the server on the SitOutRequest
    /// </summary>
    public class PlayerSitOutResponse : AbstractBluffinReponse<PlayerSitOutCommand>,IGameCommand
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public PlayerSitOutResponse(PlayerSitOutCommand command)
            : base(command)
        {
        }

        /// <summary>
        /// The id of the table where this command needs to go
        /// </summary>
        public int TableId
        {
            get { return Command.TableId; }
        }
    }
}

File: Python/BluffinMuffin.Protocol/game/player_sit_out_response.py
from abstract_game_command import AbstractGameResponse
from game.player_sit_out_command import PlayerSitOutCommand

__author__ = 'ericmas001@gmail.com'



class PlayerSitOutResponse(AbstractGameResponse):
    def __init__(self, obj):
        super().__init__(obj, PlayerSitOutCommand(obj['Command']))